user/test-super/com/google/gwt/dev/jjs/super/com/google/gwt/dev/jjs/test/Java17Test.java

```diff
@@ -41,17 +41,54 @@ public interface TextBlock {
   }
 
   public sealed class Shape permits Square, Circle {
+    public static int count = 0;
+
+    public static Shape returnAndIncrement(Shape shape){
```

Comment:
```suggestion
    public static Shape returnAndIncrement(Shape shape) {
```

merging up to date will probably catch this with the checkstyle running on java8+ tests now...